Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -3411,6 +3429,52 @@ def download_report_command(client: Client, report_id: str, instance_id: str, na
     )
 
 
+def get_asset_tags_command(client: Client, asset_id: str) -> CommandResults | list[CommandResults]:
+    """
+    Retrieve tags associated to an asset.
+
+    Args:
+        client (Client): Client to use for API requests.
+        asset_id (str): ID of the asset to retrieve information about.
+    """
+    tags = []
+
+    try:
+        tag_raw_data = client.get_asset_tags(asset_id)
+
+    except DemistoException as e:
+        if e.res is not None and e.res.status_code is not None and e.res.status_code == 404:
+            return CommandResults(readable_output="Asset not found.")
+    for tag in tag_raw_data:
+        tag_output = generate_new_dict(
+            data=tag,
+            name_mapping={
+                "type": "type",
+                "riskModifier": "risk_modifier",
+                "name": "name",
+                "created": "created_time",
```

Comment:
```suggestion
                "type": "Type",
                "riskModifier": "Risk Modifier",
                "name": "Name",
                "created": "Created Time",
```

Reply:
@adi88d This will break the JSON Output. The existing `generate_new_dict` help function takes in a `name_mapping` the values on the left are used as name values for the keys set to `tag_output` which is set in incident/alert context.